src/_backend_agg.h

```diff
@@ -488,7 +488,7 @@ RendererAgg::draw_path(GCAgg &gc, PathIterator &path, agg::trans_affine &trans,
 
     transformed_path_t tpath(path, trans);
     nan_removed_t nan_removed(tpath, true, path.has_curves());
-    clipped_t clipped(nan_removed, clip, width, height);
```

Comment:
Don't we only want to skip this if the path has a close command, not just _any_ code points.